### PR TITLE
Images from nightly

### PR DESCRIPTION
**Pull request by doutriaux1**
@downiec the PR for docker images

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -111,7 +132,8 @@ workflows:
   version: 2
   jupyter_vcdat:
     jobs:
-      - linux_jupyter_vcdat
+      #- linux_jupyter_vcdat
```
Comment (doutriaux1) [inline .circleci/config.yml]: don't forget to put back on when publishing works

File: .circleci/config.yml [hunk 2]
```diff
@@ -82,7 +82,28 @@ aliases:
...
+      docker login -u cdoutrix -p $DOCKER_PASSWORD
+      docker push cdoutrix/vcdat:nightly
+
 jobs:
+  publish:
+    docker:
+      - image: jasonb87/nimbus-basic
+    steps:
+      - checkout
+      - run: *npm_publish
```
Comment (doutriaux1) [inline .circleci/config.yml:107]: @downiec we need to publish to npm first. Images not built from an npm repo will not work on shifter.

File: kube/Dockerfile.extension [hunk 3]
```diff
@@ -1,16 +1,8 @@
-FROM cdoutrix/vcdat:lab-extensions
+FROM cdoutrix/vcdat_labextensions:latest
```
Comment (downiec) [inline kube/Dockerfile.extension]: This can be updated to cdat/vcdat_labextensions:latest

File: kube/Dockerfile.labextensions [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM cdoutrix/vcdat:core
+FROM cdoutrix/vcdat_core:latest
```
Comment (downiec) [inline kube/Dockerfile.labextensions]: Should be: FROM cdat/vcdat_core:latest

File: kube/Dockerfile.labextensions [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
 
 RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager && jupyter labextension install @jupyter-widgets/jupyterlab-sidecar && jupyter labextension install @jupyterlab/github && pip install jupyterlab_github && jupyter serverextension enable --sys-prefix jupyterlab_github && pip install nbgitpuller && jupyter serverextension enable --py nbgitpuller --sys-prefix && jupyter labextension install @jupyterlab/hub-extension && rm -rf node_modules && rm -rf ~/.cache/pip
```
Comment (downiec) [inline kube/Dockerfile.labextensions]: Need to add the tutorial manager extension I made: && jupyter labextension install tutorial-manager-extension

File: .circleci/config.yml [hunk 6]
```diff
@@ -82,7 +82,28 @@ aliases:
...
+  - &npm_publish
+    name: npm_publish
+    command: |
+      npm set //registry.npmjs.org/:_authToken=$NPM_TOKEN
+      npm publish --tag nightly
+
+  - &docker_publish
+    name: docker_publish
+    command: | 
+      docker build -f kube/Dockerfile.extension --no-cache --tag=cdoutrix/vcdat:nightly .
```
Comment (downiec) [inline .circleci/config.yml]: --tag=cdat/vcdat:nightly .

File: .circleci/config.yml [hunk 7]
```diff
@@ -82,7 +82,28 @@ aliases:
...
+      docker login -u cdoutrix -p $DOCKER_PASSWORD
+      docker push cdoutrix/vcdat:nightly
```
Comment (downiec) [inline .circleci/config.yml]: docker push cdat/vcdat:nightly

**Review by downiec — CHANGES_REQUESTED** [review]
I'm having trouble finding out why this PR doesn't pass the tests, but it seems the the npm_publish step causes issues because of missing node_modules.
'jupyter-vcdat@2.1.0 clean /home/jovyan/project
> rimraf lib

sh: 1: rimraf: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! jupyter-vcdat@2.1.0 clean: `rimraf lib`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the jupyter-vcdat@2.1.0 clean script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
'
IF this could be fixed, there's also minor updates needed to reference 'cdat' docker org, instead of personal account.

File: .circleci/config.yml (hunk 1 above)
Comment (downiec) [reply .circleci/config.yml]: Done

File: .circleci/config.yml (hunk 2 above)
Comment (downiec) [reply .circleci/config.yml:107]: Done

File: kube/Dockerfile.extension (hunk 3 above)
Comment (downiec) [reply kube/Dockerfile.extension]: Merged into single docker file.

File: kube/Dockerfile.labextensions (hunk 5 above)
Comment (downiec) [reply kube/Dockerfile.labextensions]: Done and merged into single docker file.

File: .circleci/config.yml (hunk 6 above)
Comment (downiec) [reply .circleci/config.yml]: Done

File: .circleci/config.yml (hunk 7 above)
Comment (downiec) [reply .circleci/config.yml]: Updates to config.yml have been performed.

**Review by downiec — APPROVED** [review]
Made the changes that I requested. Circleci workflow should work now.

**Review by muryanto1 — APPROVED** [review]

**Pull request merged**